openvela/media: enable media

Renjianguang-mi opened the pull request:
1. Enable media framework component and related subcomponents;
2. Enable ffmpeg and configure ffmpeg subcomponents;
3. Enable various codec libraries.

*Note: Please adhere to [Contributing Guidelines](https://github.com/open-vela/docs/blob/dev/CONTRIBUTING.md).*

## Summary

*Update this section with information on why change is necessary,
 what it exactly does and how, if new feature shows up, provide
 references (dependencies, similar problems and solutions), etc.*

## Impact

*Update this section, where applicable, on how change affects users,
 build process, hardware, documentation, security, compatibility, etc.*

## Testing

*Update this section with details on how did you verify the change,
 what Host was used for build (OS, CPU, compiler, ..), what Target was
 used for verification (arch, board:config, ..), etc. Providing build
 and runtime logs from before and after change is highly appreciated.*

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/open-vela/vendor_openvela?pullRequest=1) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/open-vela/vendor_openvela?pullRequest=1) before we can accept your contribution.<br/><hr/>**v-wangshihang** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/open-vela/vendor_openvela?pullRequest=1) it.</sub>

GUIDINGLI commented:
@w-shihang 
Please add description to this PR

GUIDINGLI commented:
Please re-run the demo

Renjianguang-mi committed:
openvela/media: enable media

1. Enable media framework component and related subcomponents;
2. Enable ffmpeg and configure ffmpeg subcomponents;
3. Enable various codec libraries.

Signed-off-by: renjianguang <renjianguang@xiaomi.com>
